snap/info.go

```diff
@@ -136,7 +136,7 @@ type Info struct {
 	Prices  map[string]float64 `yaml:"prices,omitempty" json:"prices,omitempty"`
 	MustBuy bool
 
-	Broken bool
+	Broken string
```

Comment:
While this is nice perhaps it should be an object with String() rather than a string. I wonder how this will work later with i18n.